packages/xrpl/src/utils/signPaymentChannelClaim.ts

```diff
@@ -1,5 +1,5 @@
-import binary from 'ripple-binary-codec'
-import keypairs from 'ripple-keypairs'
+import { encodeForSigningClaim } from 'ripple-binary-codec'
+import * as keypairs from 'ripple-keypairs'
```

Comment:
nit: Would this be better as a series of explicit imports like we've been doing for other imports rather than a * import?

Reply:
Agreed

Reply:
Fixed